WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift

```diff
@@ -94,7 +94,14 @@ class AppSettingsViewController: UITableViewController {
             editorRows.append(nativeEditor)
         }
 
-        let aboutHeader = NSLocalizedString("Other", comment: "Link to About section (contains info about the app)")
+        let usageTrackingHeader = NSLocalizedString("Usage Statistics", comment: "App usage data settings section header")
+        let usageTrackingRow = SwitchRow(
+            title: NSLocalizedString("Send Statistics", comment: "Label for switch to turn on/off sending app usage data"),
+            value: WPAppAnalytics.isTrackingUsage(),
+            onChange: usageTrackingChanged())
+        let usageTrackingFooter = NSLocalizedString("Automatically send usage statistics to help us improve WordPress for iOS", comment: "App usage data settings section footer describing what the setting does.")
+
+        let otherHeader = NSLocalizedString("Other", comment: "Link to About section (contains info about the app)")
```

Comment:
Agreed ... I technically didn't change this except for the constant name 🤣. Other is sort of a dumb section label anyway.